### PR TITLE
add note to CHANGELOG.md about Version parser updates

### PR DESCRIPTION
**Pull request by ntrepid8**
This PR adds a note to the CHANGELOG about the Version parser updates. Also see: #6155

### CODE REVIEW

**Comment by ntrepid8** [comment]
This is my first elixir PR so apologies but I'm not sure what to do about this error from the CI system:
```
Specify a project or solution file. The directory does not contain a project or solution file.
```

I looked through the section on contributing but on my initial look I didn't see anything that seemed relevant. Can someone point me in the right direction?

File: CHANGELOG.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ This version includes changes that make Elixir fully compatible with Erlang OTP
 
   * [Kernel] Fix code generation when non-binary bitstrings are in AST
   * [Record] Properly escape fields passed to `defrecord`
+  * [Version] Update parser to reject leading zeros according to the SemVer spec
```
Comment (ericmj) [inline CHANGELOG.md]: I think it can be changed to: ~~Update parser to~~ reject leading zeros according to the SemVer spec

**Comment by ericmj** [comment]
You can ignore the errors from appveyor for now.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**